lib/mix/lib/mix/tasks/app.tree.ex

```diff
@@ -0,0 +1,70 @@
+defmodule Mix.Tasks.App.Tree do
+  use Mix.Task
+
+  @shortdoc "Analyses the current Mix project and prints dependency tree on terminal."
```

Comment:
Probably we need to remove terminal wording, it's just a particular case when `Mix.shell` is set to `IO`.